log_injection/irzaip_ChippyBot/wa.py

```
@@ -0,0 +1,478 @@
+    chars = string.ascii_lowercase + string.digits
+    return ''.join(random.choice(chars) for _ in range(10))
+ 
+def send_to_phone(user_number: str, bot_number: str, message: str):
+    """send langsung ke WA, tapi ke *user_number*, bukan ke bot_number"""
+    message = {
+        "message": message, # Replace with your message text
+        "from": bot_number, # Replace with the sender number
+        "to": user_number # Replace with out bot number
+    } # type: ignore
+
+    response = requests.post(whatsapp_web_url, json=message)
```

Ruff (0.13.3)

Probable use of `requests` call without timeout

(S113)